src/preferences/dialog/dlgprefdeckdlg.ui

```diff
@@ -152,19 +156,39 @@ CUP mode:
         </property>
        </widget>
       </item>
-      <item row="2" column="0">
-       <widget class="QLabel" name="labelTimeDisplay">
+      <item row="4" column="1" colspan="2">
+       <widget class="QCheckBox" name="checkBoxDisallowLoadToPlayingDeck">
         <property name="text">
-         <string>Time Format</string>
+         <string>Do not load tracks into playing decks</string>
         </property>
        </widget>
       </item>
-      <item row="2" column="1">
-       <widget class="QComboBox" name="comboBoxTimeFormat"/>
+      <item row="5" column="0">
+       <widget class="QLabel" name="labelCloneDeckOnLoadDoubleTap">
+        <property name="text">
+         <string>Clone Deck</string>
+        </property>
+        <property name="buddy">
+         <cstring>checkBoxCloneDeckOnLoadDoubleTap</cstring>
+        </property>
+       </widget>
+      </item>
+      <item row="5" column="1" colspan="2">
+       <widget class="QCheckBox" name="checkBoxCloneDeckOnLoadDoubleTap">
+        <property name="toolTip">
+         <string>Create a playing clone of the first playing deck Mixxx finds
+         by double-tapping a Load button on a controller or keyboard.
+         You can always drag-and-drop tracks on screen to clone a deck.</string>
```

Comment:
Remove leading spaces. They are included in the tooltips.

Reply:
done.